scripts/upgrades/upgrade-helper.ts

```diff
@@ -0,0 +1,40 @@
+import { Interface } from '@ethersproject/abi';
+import { ContractTransaction } from '@ethersproject/contracts';
+import { Signer } from 'ethers';
+import { ethers } from 'hardhat';
+import { ERC1538Query, ERC1538Query__factory, ERC1538Update__factory } from '../../typechain';
+
+function encodeModuleProxyUpdate(ModuleInterface: Interface, moduleAddress: string) {
+    const moduleFunctions = Object.keys(ModuleInterface.functions).map((tx) => tx + ';');
+    moduleFunctions.forEach((func) => {
+        console.log(`- ${func}`);
+    });
+    const moduleProxyUpdateData = ERC1538Update__factory.createInterface().encodeFunctionData(
+        'updateContract',
+        [moduleAddress, moduleFunctions.join(''), ''],
+    );
+    return moduleProxyUpdateData;
+}
+
+async function printBlockTime() {
+    const latestBlock = await ethers.provider.getBlock('latest');
+    const blockNumber = latestBlock.number;
+    const blockTimestamp = latestBlock.timestamp;
+    const blockDate = new Date(blockTimestamp * 1000);
+    console.log(`Block#${blockNumber}: ${blockDate} (timestamp:${blockTimestamp})`);
+}
+
+async function printFunctions(erc1538ProxyAddress: string, owner: Signer) {
```

Comment:
```suggestion
async function printFunctionsCount(erc1538ProxyAddress: string) {
```
With a random signer
```typescript
const [ anyone ] = await ethers.getSigners();
```

Reply:
Just added missing function logging part thanks! (I also updated new output in PR description)